xla/stream_executor/cuda/cuda_blas_lt.cc

```diff
@@ -315,11 +318,17 @@ auto BlasLt::GetMatmulPlan(const gpu::GemmConfig& cfg,
                                           cfg.compute_precision));
   }
 
+  // For FP8 matmuls, there are two options available: fast
+  // accumulation(PrecisionConfig.Precision.DEFAULT) and
+  //  higher precision accumulation (PrecisionConfig.Precision.HIGHEST).
```

Comment:
You don't mention the HIGHER case. I would phrase this as:

> FP8 matmuls have a fast accumulation mode that is less precise than the default accumulation mode. Use the fast accumulation mode if the compute precision is DEFAULT.

Reply:
Added a test in gemm_rewrite_test.cc